modules/virt-about-datavolumes.adoc

```diff
@@ -9,13 +9,12 @@
 // * virt/virtual_machines/cloning_vms/virt-cloning-vm-disk-into-new-datavolume-block.adoc
 // * virt/virtual_machines/importing_vms/virt-importing-virtual-machine-images-datavolumes.adoc
 // * virt/virtual_machines/importing_vms/virt-importing-virtual-machine-images-datavolumes-block.adoc
+// * virt/virtual_machines/virt-deleting-datavolumes.adoc
 
 :_content-type: CONCEPT
-[id="virt-about-datavolumes_{context}"]
-= About data volumes
+[id="virt-about-datavolumes-and-pvcs_{context}"]
+= About data volumes and persistent volume claims (PVCs)
 
-`DataVolume` objects are custom resources that are provided by the Containerized
-Data Importer (CDI) project. Data volumes orchestrate import, clone, and upload
-operations that are associated with an underlying persistent volume claim (PVC).
-Data volumes are integrated with {VirtProductName}, and they prevent a virtual machine
-from being started before the PVC has been prepared.
+Data volumes are used by {VirtProductName} to orchestrate import, clone, and upload operations that are associated with an underlying PVC, which the data volume creates. Once the PVC is created, you use the PVC to manage the virtual machine disks in your cluster. The data volume that you used to create the PVC is no longer needed.
+
+For example, to verify the size of a virtual machine disk, you examine the PVC specification, instead of the data volume. The data volume's spec contains only the initial allocation given to the PVC. The DV will not be updated after the data volume has been initially created.
```

Comment:
The DV will not be updated after the data volume has been initially created. -> The DV is no longer updated after it's initial creation.